{"schemaVersion":{"patch":0,"major":0,"minor":3},"sections":[],"hierarchy":{"paths":[["doc:\/\/Passage\/documentation\/Passage","doc:\/\/Passage\/documentation\/Passage\/OAuth2API","doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"requestUriNotSupported","kind":"identifier"}]}]}],"kind":"symbol","metadata":{"role":"symbol","externalID":"s:7Passage9OAuth2APIC25ModelError_oauth2CallbackO22requestUriNotSupportedyA2EmF","symbolKind":"case","title":"OAuth2API.ModelError_oauth2Callback.requestUriNotSupported","modules":[{"name":"Passage"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"requestUriNotSupported","kind":"identifier"}],"roleHeading":"Case"},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback\/requestUriNotSupported"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/passage\/oauth2api\/modelerror_oauth2callback\/requesturinotsupported"]}],"references":{"doc://Passage/documentation/Passage/OAuth2API/ModelError_oauth2Callback/requestUriNotSupported":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"requestUriNotSupported"}],"kind":"symbol","type":"topic","abstract":[],"identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback\/requestUriNotSupported","title":"OAuth2API.ModelError_oauth2Callback.requestUriNotSupported","url":"\/documentation\/passage\/oauth2api\/modelerror_oauth2callback\/requesturinotsupported","role":"symbol"},"doc://Passage/documentation/Passage/OAuth2API/ModelError_oauth2Callback":{"role":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ModelError_oauth2Callback"}],"abstract":[{"type":"text","text":"enum for parameter error"}],"title":"OAuth2API.ModelError_oauth2Callback","kind":"symbol","url":"\/documentation\/passage\/oauth2api\/modelerror_oauth2callback","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"ModelError_oauth2Callback"}],"identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback"},"doc://Passage/documentation/Passage":{"kind":"symbol","type":"topic","abstract":[{"text":"Use Passage to add exclusively Passwordless authentication to your iOS Application.","type":"text"}],"url":"\/documentation\/passage","title":"Passage","identifier":"doc:\/\/Passage\/documentation\/Passage","role":"collection"},"doc://Passage/documentation/Passage/OAuth2API":{"type":"topic","url":"\/documentation\/passage\/oauth2api","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"OAuth2API"}],"identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API","abstract":[],"fragments":[{"text":"class","kind":"keyword"},{"text":" ","kind":"text"},{"text":"OAuth2API","kind":"identifier"}],"title":"OAuth2API","role":"symbol"}}}